mqcloudv1/mqcloud_v1_integration_test.go

```diff
@@ -725,13 +727,127 @@ var _ = Describe(`MqcloudV1 Integration Tests`, func() {
+				Expect(response.StatusCode).To(Equal(200))
+				Expect(virtualPrivateEndpointGatewayDetailsCollection).ToNot(BeNil())
+				allResults = append(allResults, virtualPrivateEndpointGatewayDetailsCollection.VirtualPrivateEndpointGateways...)
+
+				listVirtualPrivateEndpointGatewaysOptions.Start, err = virtualPrivateEndpointGatewayDetailsCollection.GetNextStart()
+				Expect(err).To(BeNil())
+				if listVirtualPrivateEndpointGatewaysOptions.Start == nil {
+
+					break
+				}
+				fmt.Println("START=" + *listVirtualPrivateEndpointGatewaysOptions.Start)
+			}
+			fmt.Fprintf(GinkgoWriter, "Retrieved a total of %d item(s) with pagination.\n", len(allResults))
+		})
+		It(`ListVirtualPrivateEndpointGateways(listVirtualPrivateEndpointGatewaysOptions *ListVirtualPrivateEndpointGatewaysOptions) using VirtualPrivateEndpointGatewaysPager`, func() {
+			listVirtualPrivateEndpointGatewaysOptions := &mqcloudv1.ListVirtualPrivateEndpointGatewaysOptions{
+				ServiceInstanceGuid: core.StringPtr(config["SERVICE_INSTANCE_CAPACITY_GUID"]),
+				TrustedProfile:      core.StringPtr(config["TRUSTED_PROFILE"]),
+				Limit:               core.Int64Ptr(int64(10)),
+			}
+
+			// Test GetNext().
+			listVirtualPrivateEndpointGatewaysOptions.TrustedProfile = nil
+			pager, err := mqcloudService.NewVirtualPrivateEndpointGatewaysPager(listVirtualPrivateEndpointGatewaysOptions)
+			Expect(err).To(BeNil())
+			Expect(pager).ToNot(BeNil())
+
+			var allResults []mqcloudv1.VirtualPrivateEndpointGatewayDetails
+			for pager.HasNext() {
+				nextPage, err := pager.GetNext()
+				Expect(err).To(BeNil())
+				Expect(nextPage).ToNot(BeNil())
+				allResults = append(allResults, nextPage...)
+			}
+
+			// Test GetAll().
+			pager, err = mqcloudService.NewVirtualPrivateEndpointGatewaysPager(listVirtualPrivateEndpointGatewaysOptions)
+			Expect(err).To(BeNil())
+			Expect(pager).ToNot(BeNil())
+
+			allItems, err := pager.GetAll()
+			Expect(err).To(BeNil())
+			//		Expect(allItems).ToNot(BeNil())
+
+			Expect(len(allItems)).To(Equal(len(allResults)))
+			fmt.Fprintf(GinkgoWriter, "ListVirtualPrivateEndpointGateways() returned a total of %d item(s) using VirtualPrivateEndpointGatewaysPager.\n", len(allResults))
+		})
+	})
+
+	Describe(`GetVirtualPrivateEndpointGateway - Display the information for a specific virtual private endpoint gateway`, func() {
+		BeforeEach(func() {
+			shouldSkipTest()
+		})
+		It(`GetVirtualPrivateEndpointGateway(getVirtualPrivateEndpointGatewayOptions *GetVirtualPrivateEndpointGatewayOptions)`, func() {
+			getVirtualPrivateEndpointGatewayOptions := &mqcloudv1.GetVirtualPrivateEndpointGatewayOptions{
+				ServiceInstanceGuid:               core.StringPtr(config["SERVICE_INSTANCE_CAPACITY_GUID"]),
+				VirtualPrivateEndpointGatewayGuid: core.StringPtr(config["VIRTUAL_PRIVATE_ENDPOINT_GATEWAY_GUID"]),
+				TrustedProfile:                    core.StringPtr(config["TRUSTED_PROFILE"]),
+			}
+
+			getVirtualPrivateEndpointGatewayOptions.TrustedProfile = nil
```

Comment:
Seeing this optional parameter `TrustedProfile` has been assigned value`config["TRUSTED_PROFILE"]` and removed right after setting it in 4 instances under integration test. 

Examples test looks fine.

Would be good to not assign the value, or dont set it to nil, if its works fine

Reply:
Generated code is like that, you can see offset for queuemanager resource

Reply:
Will be good to double check, because i generated the code locally with the same sdk generator version for the review. 

it seems like manually added line of code.

Reply:
Yes, I have added these lines of code because the trusted profile is not required as we can access all the resources

Reply:
Ok. Just to know, is there any harm if we remove that added line by passing this param with some value.

Because it will be really good to minimise the manual tweaking to the file for the future easiness for generation.